### DISCUSSION TITLE
deepin-editor: 希望支持最近打开文件列表(历史记录)

### DISCUSSION THREAD

**Discussion by qcxiang** [linuxdeepin/developer-center#2849] [Features Request & Ideas | 特性请求 & 头脑风暴]
希望支持最近打开文件列表，或者叫最近打开的记录这样的功能。要不每次都需要重头开始找需要打开的文件。

**Comment by YutianGe** [comment]
Thank you for your advice

**Comment by guoshaoyu** [comment]
> 希望支持最近打开文件列表，或者叫最近打开的记录这样的功能。要不每次都需要重头开始找需要打开的文件。

感谢您的建议，我们会合理考虑此功能。
